src/charactersheet/viewmodels/dm/campaign_maps_and_images/form.html

```diff
@@ -0,0 +1,94 @@
+<form data-bind="submit: submit, with: entity">
+  <card-submit-actions params="{reset: $component.reset}"></card-submit-actions>
+  <!-- ko if: $component.addForm -->
+  <div class="h4 collapsable-card-title">New Map or Image</div>
+  <!-- /ko -->
+  <div class="row">
+    <div class="form-group col-xs-12">
+      <label>Name<i class="required"></i></label>
+      <input
+        type="text"
+        class="form-control"
+        name="name"
+        placeholder="World Map"
+        data-bind="
+          textInput: name,
+          hasFocus: $component.formElementHasFocus,
+          event: { blur: $component.reviewInput, invalid: $component.invalidate },
+          attr: { ...$component.validation.name },
+        "
+      />
+    </div>
+    <div class="form-group col-xs-12">
+      <div class="d-flex justify-content-between">
+        <label class="d-flex-1">
+          Image URL<i class="required"></i></label>
+        </label>
+        <div class="d-flex-1 text-right text-muted">
+          <i
+            class="fa fa-info-circle"
+            title="This field will be sent to the players via the exhibit."
+          ></i>&nbsp; Exhibit
+        </div>
+      </div>
+      <input
```

Comment:
This is unrelated, but I was able to reproduce something I found in another PR.
Using `picsum.photos`, create an `moi` with size `200`. Once that's created, edit the same item and change the size to `1200`. The image will overflow.
![image](https://user-images.githubusercontent.com/7286387/148004835-198197bf-75f3-42ea-ab26-857fd389ee3d.png)

Reply:
This issue is due to the fact that the cards are assigned a fixed size before the image is fully loaded. I will log it as a bug and fix it in another PR.